deploy/helm/templates/hub/servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.hub.monitoring.enabled }}
```

Comment:
Helm recommends that we check to make sure the parents exist to avoid a null pointer error.

{{- if .Values.hub 
{{- if .Values.monitoring 
{{- if .Values.enabled 

{{- end -}}
{{- end -}}
{{- end -}}


or more simply:

Wrap each nullable level with parentheses () as a go-like pipeline operator.

{{ ((.Values.foo).bar) }}

Or

{{ if ((.Values.foo).bar) }}
{{ .Values.foo.bar }}
{{ end }}

Reference: https://stackoverflow.com/questions/59795596/helm-optional-nested-variables